app/models/verification/base.rb

```diff
@@ -1,24 +1,24 @@
 # frozen_string_literal: true
+
+# Basic verification class
+# No bed verification, just takes a list of source plates
+# and records that the action has been perfromed.
 class Verification::Base
   include ActiveModel::Validations
-
-  attr_accessor :instrument_barcode
-  attr_accessor :scanned_values
-  attr_accessor :api
-
```

Comment:
The getters are redefined in the class to fetch the attributes from a hash in @attributes, the setters are unused, and would show the wrong behaviour anyway. (Setting the instance variables) So removed.